src/components/HomepageHeader/HomepageHeader.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Logo from "../Logo/Logo.js";
+import SearchBar from "../SearchBar/SearchBar.js";
```

Comment:
Just a small proposal which I personally like, and usually we use it on projects. You have own folder for each component, so you could create index.js file inside with:
`export { default } from './SearchBar';`
then just import it like this:
`import SearchBar from "../SearchBar;`
and you also don't need to add extension 'js' for file.
But again, It's just a proposal, and you could do it as you prefer.

Reply:
A good approach, I will definitely use it in my next projects